Makefile.PL

```diff
@@ -86,6 +84,15 @@ unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
 delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
   unless eval { ExtUtils::MakeMaker->VERSION(6.52) };
 
+if ( $^O !~ /win/i ) {
+	$WriteMakefileArgs{PREREQ_PM}{'Alien::Build::Plugin::Build::Autoconf'} = $FallbackPrereqs{'Alien::Build::Plugin::Build::Autoconf'} = '0.04';
```

Comment:
These plugins `Build::Autoconf` and `Build::CMake` should really be configure requires.  Even though we will only use one or the other, we need to have the one that we need before running `perl Makefile.PL`.  I think `Build::Autoconf` was always part of AB Core, but `Build::CMake` wasn't.  It should only matter if someone is using an older version of AB, but it is better to add the configure requires in `dist.ini`.  Normally the `[AlienBuild]` plugin handles this, but since the prereqs depend on the platform that you build it on we need to do it manually in the `dist.ini`.